From: Remi Collet <remi@remirepo.net>
Date: Thu, 30 Jul 2026 09:07:49 +0200
Subject: NEWS from 8.2.33

(cherry picked from commit 11b947ce14c7fcc77adc740349add4c5c8967a62)
(cherry picked from commit 9aa1f1c43308560bb5dc97cbf8995c66108aeef0)
---
 NEWS | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/NEWS b/NEWS
index cf9a158..6e0a091 100644
--- a/NEWS
+++ b/NEWS
@@ -6,6 +6,17 @@ Backported from 8.2.33
 - Date:
   . Fixed leak on double DatePeriod::__construct() call. (ilutov)
 
+- GD:
+  . Upgrade libgd. (CVE-2026-9672) (Pierre Joye)
+
+- PGSQL:
+  . Fixed GHSA-7qpv-r5mr-78m4 (SQL injection via E'...' backslash breakout).
+    (CVE-2026-17543) (ilutov)
+
+- Phar:
+  . Fixed GHSA-vc5h-9ppw-p5f3 (Crash via recursive symlinks). (CVE-2026-7260)
+    (Jakub Zelenka)
+
 Backported from 8.2.32
 
 - OpenSSL:
